OpenRA.Game/Platform.cs

```diff
@@ -94,12 +94,25 @@ static string GetSupportDir()
 			return dir + Path.DirectorySeparatorChar;
 		}
 
-		public static string GameDir { get { return AppDomain.CurrentDomain.BaseDirectory; } }
+		public static string GameDir
+		{
+			get
+			{
+				var gameDir = AppDomain.CurrentDomain.BaseDirectory;
+
+				// BUG: In Game.cs we get GameDir with trailing DirectorySeparator, but not in PlatformTest.cs
+				if (!gameDir.EndsWith(Path.DirectorySeparatorChar.ToString()))
+				{
+					gameDir += Path.DirectorySeparatorChar;
+				}
```

Comment:
There is https://github.com/OpenRA/OpenRA/blob/bleed/Settings.StyleCop, but it doesn't catch all. I don't know if it isn't up to date or it doesn't support all our conventions.